utils/gyb_syntax_support/AttributeNodes.py

```diff
@@ -202,10 +206,9 @@
          ]),
 
     # differentiation-params-clause ->
-    #     'wrt' ':' (differentiation-param | differentiation-params) ','?
+    #     'wrt' ':' (differentiation-param | differentiation-params)
```

Comment:
The `wrt` clause can have a trailing comma in `@differentiable` attribute, but not in `@differentiating` attribute.
Removing the trailing comma from the `DifferentiationParamsClause` syntax node is correct/necessary to unify parsing.